#   Source: https://bugs.launchpad.net/exaile/+bug/421611/comments/1 
# Upstream: https://bugs.launchpad.net/exaile/+bug/421611/comments/2 (fixed in r2439 / .31)
#   Reason: "make install" runs compilepo.sh. This step should happen as part of "make all", not "make install"

diff -ur exaile-0.3.0.orig/Makefile exaile-0.3.0/Makefile
--- exaile-0.3.0.orig/Makefile  2009-08-27 22:24:25.000000000 +0300
+++ exaile-0.3.0/Makefile       2009-08-27 22:25:14.000000000 +0300
@@ -5,7 +5,7 @@
 EXAILELIBDIR = $(DESTDIR)$(PREFIX)$(LIBINSTALLDIR)/exaile
 EXAILESHAREDIR = $(DESTDIR)$(PREFIX)/share/exaile
 
-all: compile
+all: compile locale
 	@echo "Ready to install..."
 
 compile:
@@ -46,7 +46,7 @@
 	rm -f $(DESTDIR)$(PREFIX)/share/pixmaps/exaile.png
 	cd plugins && make uninstall && cd ..
 
-install: install-target locale install-locale
+install: install-target install-locale
 
 install_no_locale: install-target
 

